src/mc_tools.jl

```diff
@@ -289,10 +261,16 @@ None modifies `samples` in place
 """
 function mc_sample_path!(mc::MarkovChain, samples::Array)
     # ensure floating point input for Categorical()
-    p       = convert(Matrix{Float64}, mc.p)
+    p       = convert(Matrix{Float64}, mc.p) 
     dist    = [Categorical(vec(p[i, :])) for i=1:n_states(mc)]
     for t=2:length(samples)
         samples[t] = rand(dist[samples[t-1]])
     end
     nothing
 end
+
+function simulate(mc::MarkovChain, 
+    init::Vector = vcat(1,zeros(n_states(mc)-1)), 
+    sample_size=1000)
+    return mc_sample_path(mc,init,sample_size)
+end
```

Comment:
Another picky style thing -- have all function arguments that are on different lines start at the same column as the first function argument like we do [here](https://github.com/QuantEcon/QuantEcon.jl/blob/cf59a18c90c56f9a55a4157771306a4ebb8f9a76/src/mc_tools.jl#L240-L242) (and all other places in the library)

Sorry about these irritating requests. I really do need to write up a style guide so people besides me know what the style guidelines are. I'll open up an issue to remind myself.